cli/cmd/generate.go

```diff
@@ -231,19 +231,38 @@ func findGenerateTargets(args []string, logger *slog.Logger) ([]string, error) {
 			return nil
 		})
 		if err != nil {
-			return nil, fmt.Errorf("walk %s: %w", path, err)
+			return nil, nil, fmt.Errorf("walk %s: %w", path, err)
 		}
 	}
 	if len(paths) == 0 {
-		return nil, fmt.Errorf("no .yml/.yaml files found")
+		return nil, nil, fmt.Errorf("no .yml/.yaml files found")
 	}
 
+	cmPaths := filterForConfigMaps(paths, logger)
+
 	paths = filterNonCoCoRuntime("contrast-cc", paths, logger)
 	if len(paths) == 0 {
-		return nil, fmt.Errorf("no .yml/.yaml files with 'contrast-cc' runtime found")
+		return nil, nil, fmt.Errorf("no .yml/.yaml files with 'contrast-cc' runtime found")
 	}
 
-	return paths, nil
+	return paths, cmPaths, nil
+}
+
+func filterForConfigMaps(paths []string, logger *slog.Logger) []string {
+	var filtered []string
+	for _, path := range paths {
+		data, err := os.ReadFile(path)
+		if err != nil {
+			logger.Warn("read file", "path", path, "err", err)
+			continue
+		}
+		if !bytes.Contains(data, []byte("ConfigMap")) {
```

Comment:
Can we do the proper check here - `UnmarshalUnstructuredK8SResource` and then compare group, version, kind?